media/js/styleguide/docs/send-to-device.js

```diff
@@ -6,5 +6,10 @@ $(function() {
     'use strict';
 
     var form = new Mozilla.SendToDevice();
+
+    form.geoCallback = function(countryCode) {
+        alert('The country code is: ' + countryCode);
```

Comment:
I wanted to put *something* in that function (and our linter wont allow `console.log` statements), but I suppose an `alert` is rather annoying. Will remove.